mopidy/mpd/protocol/stored_playlists.py

```diff
@@ -201,7 +245,17 @@ def rename(context, old_name, new_name):
 
         Renames the playlist ``NAME.m3u`` to ``NEW_NAME.m3u``.
     """
-    raise exceptions.MpdNotImplemented  # TODO
+    uri = context.lookup_playlist_uri_from_name(old_name)
+    playlist = uri is not None and context.core.playlists.lookup(uri).get()
+    if not playlist:
+        raise exceptions.MpdNoExistError('No such playlist')
+
+    # Create copy of the playlist and remove original
+    copy = context.core.playlists.create(new_name).get()
```

Comment:
Provide the uri scheme of the original here or else you might get a playlist of the wrong type.